linsolve/linsolve.py

```diff
@@ -250,6 +251,7 @@ def __init__(self, data, wgts={}, sparse=False, **kwargs):
         Returns:
```

Comment:
I think the description of the kwarg `sparse` is no longer correct and we should update it to clarify that while the A matrices are represented sparsely, the AtA matrices are not.